apps/web/composables/useProduct/useProduct.ts

```diff
@@ -1,6 +1,12 @@
-import { categoryTreeGetters, type CategoryTreeItem, Product, type ProductParams } from '@plentymarkets/shop-api';
+import type { Product, ProductParams } from '@plentymarkets/shop-api';
 import { productGetters } from '@plentymarkets/shop-api';
-import { type UseProductReturn, UseProductState, type FetchProduct } from '~/composables/useProduct/types';
+import { toRefs } from '@vueuse/shared';
+import type { UseProductReturn, UseProductState, FetchProduct } from '~/composables/useProduct/types';
+// import { categoryTreeGetters, type CategoryTreeItem, Product, type ProductParams } from '@plentymarkets/shop-api';
+// import { productGetters } from '@plentymarkets/shop-api';
+// import { type UseProductReturn, UseProductState, type FetchProduct } from '~/composables/useProduct/types';
```

Comment:
**todo**: Delete imports if they're no longer used.

Reply:
@ksted omw, i forgot to remove. Commented out while conflic resolving.